modules/removee.js

```diff
@@ -0,0 +1,21 @@
+import Book from './bookclass.js';
+
+const id = 0;
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const newBook = new Book(title, author, id);
+
+const removee = () => {
+  document.querySelector('.form-book').addEventListener('submit', (e) => {
+    e.preventDefault();
+    newBook.bookTitle = title.value;
+    newBook.bookAuthor = author.value;
+    newBook.bookId = id;
+    if (newBook.bookTitle && newBook.bookAuthor) {
+      Book.storeBook(newBook);
+      window.location.reload();
+    }
+  });
+};
```

Comment:
- [ ] Kindly fix the following behavior when the user adds and then removes an item.

https://user-images.githubusercontent.com/62290545/177165367-84064a67-fdfb-4ebb-84c5-126944671591.mp4